app/views/users/_edit_form.html.erb

```diff
@@ -33,7 +35,7 @@
 
       <div class="form-group col-sm-6">
         <label for="username"><%= t('users._form.username') %></label>
-        <%= f.text_field :username, { tabindex: 1, placeholder: "Username", class: 'form-control', id: 'username' } %>
+        <%= f.text_field :username, { readonly: true, tabindex: 1, placeholder: "Username", class: 'form-control', id: 'username' } %>
```

Comment:
Actually you are working on a branch which is not consistent with the current master. In the latest merge this line was removed. please merge the latest master branch into your current branch. Then you will get conflicts. You need to resolve them.